lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -677,15 +672,44 @@ struct ziti_instance_s *new_ziti_instance(const char *identifier) {
 }
 
 int init_ziti_instance(struct ziti_instance_s *inst, const ziti_config *cfg, const ziti_options *opts) {
+    FREE(inst->ztx);
     int rc = ziti_context_init(&inst->ztx, cfg);
     if (rc != ZITI_OK) {
+        ZITI_LOG(ERROR, "ziti_context_init failed: %s", ziti_errorstr(rc));
         return rc;
     }
 
     rc = ziti_context_set_options(inst->ztx, opts);
+    if (rc != ZITI_OK) {
+        ZITI_LOG(ERROR, "ziti_context_set_options failed: %s", ziti_errorstr(rc));
+        FREE(inst->ztx);
+        return rc;
+    }
+
+    rc = set_tnlr_options(inst);
+    if (rc != ZITI_OK) {
+        FREE(inst->ztx);
```

Comment:
no `ZITI_LOG` in this situation? presumably because `set_tnlr_options` does any logggin?